config/config.yaml.example

```diff
@@ -0,0 +1,2 @@
+ENV_PRODUCT: k3s
```

Comment:
add the value as variable = {product}

Reply:
isnt this supposed to be a yaml file format? we should be doing 'variable: value' in which case?

Reply:
why do you think use a var in the value is not a yaml ?

Reply:
i think we both misunderstood each other. my problem was with using "=" instead of the ":" in a yaml. - since that is not yaml formatting. got where you are coming from now 👍

Reply:
Fixed.